pkg/handler/status.go

```diff
@@ -1,14 +1,74 @@
 package handler
 
 import (
+	"context"
 	"net/http"
-
-	"github.com/sirupsen/logrus"
+	"strings"
 )
 
-func Status(w http.ResponseWriter, _ *http.Request) {
-	_, err := w.Write([]byte("OK"))
-	if err != nil {
-		logrus.Errorf("could not write response: %v", err)
+type Status struct {
+	IsAllowProm         bool `yaml:"isAllowProm" json:"isAllowProm"`
+	PromNamespacesCount int  `yaml:"promNamespacesCount" json:"promNamespacesCount"`
+	IsAllowLoki         bool `yaml:"isAllowLoki" json:"isAllowLoki"`
+	LokiNamespacesCount int  `yaml:"lokiNamespacesCount" json:"lokiNamespacesCount"`
+	IsLokiReady         bool `yaml:"isLokiReady" json:"isLokiReady"`
+	IsConsistent        bool `yaml:"isConsistent" json:"isConsistent"`
```

Comment:
If I'm right, you can remove this `IsConsistent` from the REST API because it's not used in the js code - it's re-computed there (and I'd be in favor to keep it that way, computed on frontend side only, since it has all the data to compute it).

Also, we need to see if we find false-positives of this consistency check - I'm fine to keep it for the time being, but given that metrics are configurable, I'm wondering if there could be undesired "inconsistent" triggers. We'll see